src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,8 @@
             android:theme="@style/AppTheme">
         <activity
                 android:name=".MainActivity"
-                android:theme="@style/Theme.AppCompat.Light.NoActionBar">
+                android:theme="@style/Theme.AppCompat.Light.NoActionBar"
+                android:exported="true">
```

Comment:
Not clear on that, it seems that `android:exported` is now required on some `<activity>` but maybe not all of them?